release-pipeline.yml

```diff
@@ -2168,6 +2168,8 @@ stages:
 
             - script: |
                 pip install azure-storage-blob
-                python3 releaseVersionUpdate.py "$(VERSION_CNT_SAS_URL)" "`azcopy --version`"
+                AZCOPY_AUTO_LOGIN_TYPE=AzCLI
+                ACCESS_TOKEN=$(az account get-access-token) 
```

Comment:
Do we need to pass the Access Token? Doesnt look like its being used in the script.

Reply:
I intended to have it to replace the SAS token, but they have different uses cases. So, yes, it's not needed.